Documentation/operations/upgrade.rst

```diff
@@ -336,6 +336,8 @@ Removed Options
   please use the ``--kube-version`` flag (introduced in Helm 3.6.0) instead.
 * The deprecated ``hubble-ca-cert`` configmap has been removed. Use
   ``hubble-ca-secret`` secret instead.
+* The deprecated ``azure-cloud-name`` option for cilium-operator-azure was
+  deprecated in 1.10 and is now removed.
```

Comment:
Nit: `The deprecated ... was deprecated in ...` sounds a bit redundant?

Reply:
Indeed :smile: dropped the first one.

Reply:
thank you :)